tests/storage/hotplug.go

```diff
@@ -430,6 +426,32 @@ var _ = SIGDescribe("Hotplug", func() {
 		return vm
 	}
 
+	checkNoProvisionerStorageClassPVs := func(storageClass string) {
+		sc, err := virtClient.StorageV1().StorageClasses().Get(context.Background(), storageClass, metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		if sc.Provisioner != "" && sc.Provisioner != "kubernetes.io/no-provisioner" {
+			return
+		}
+
+		// Verify we have at least 3 available file system PVs
+		pvList, err := virtClient.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		count := 0
+		for _, pv := range pvList.Items {
+			if pv.Spec.NodeAffinity == nil || pv.Spec.NodeAffinity.Required == nil || len(pv.Spec.NodeAffinity.Required.NodeSelectorTerms) == 0 || (pv.Spec.VolumeMode != nil && *pv.Spec.VolumeMode == corev1.PersistentVolumeBlock) {
```

Comment:
You will want to check that the PV storage class matches the storageClassName passed in.